### PR TITLE
Time evolution simulation with Sequencing plus demo tutorial

### PR DESCRIPTION
**Pull request by quantum-quant**
### What are the issues this pull addresses (issue numbers / links)?
Time evolution simulation with Sequencing plus demo tutorial

### Did you add tests to cover your changes (yes/no)?
no

### Did you update the documentation accordingly (yes/no)?
yes

### Did you read the CONTRIBUTING document (yes/no)?
yes

### CODE REVIEW

File: qiskit_metal/analyses/quantization/lom_time_evolution_sim.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    """
+
+    @staticmethod
+    def mapper(metal_system):
+        if metal_system.sys_type == 'TRANSMON':
+            return seq.Transmon
+        elif metal_system.sys_type in ['FLUXONIUM']:
+            return seq.Qubit
+        elif metal_system.sys_type in ['TL_RESONATOR', 'LUMPED_RESONATOR']:
+            return seq.Cavity
```
[the remark this answers is not in the document]
Comment (zlatko-minev) [reply qiskit_metal/analyses/quantization/lom_time_evolution_sim.py:20]: i.e. on the fly so we dont need to hard code it in the code here

**Review by zlatko-minev — CHANGES_REQUESTED** [review]
see class LOMtoSeqMapper comments

**Comment by priti-ashvin-shah-ibm** [comment]
> [CLA assistant check](https://cla-assistant.io/Qiskit/qiskit-metal?pullRequest=753) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Qiskit/qiskit-metal?pullRequest=753) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Qiskit/qiskit-metal?pullRequest=753) it.

@quantum-quant , there was an issue with CLA, could you sign it again?

File: qiskit_metal/analyses/quantization/lom_time_evolution_sim.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+from qiskit_metal.analyses.quantization.lom_core_analysis import CompositeSystem
+
+
+class LOMtoSeqMapper:
+    """Mapping LOM subsystem to appropriate Sequencing mode
+    """
+
+    @staticmethod
+    def mapper(metal_system):
```
[the remark this answers is not in the document]
Comment (quantum-quant) [reply qiskit_metal/analyses/quantization/lom_time_evolution_sim.py]: fixed based on feedback

File: qiskit_metal/analyses/quantization/lom_time_evolution_sim.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (quantum-quant) [reply qiskit_metal/analyses/quantization/lom_time_evolution_sim.py]: fixed based on feedback

**Review by zlatko-minev — APPROVED** [review]

**Pull request merged**